packages/dcos-launch/extra/README.md

```diff
@@ -0,0 +1,28 @@
+# dcos-launch User Guide
+## Purpose
+`dcos-launch` is intendends to provide the following:
+* Consistent Interface: launching on Azure or AWS can involve finding a suitable client API and then sifting through hundreds of specific API calls to find the combination that will allow deploying DC/OS. `dcos-launch` has the exact methods required to take a DC/OS build artifact and deploy it to an arbitrary provider with no other tools.
+* Turn-key Deployment: The only input required by dcos-launch is the [launch config](launch/sample_configs/README.md). In cases where artifacts are required to deploy DC/OS (e.g. SSH keys), `dcos-launch` provids helpers to automatically create and cleanup those extra artifacts
```

Comment:
Does the `launch/sample_configs/README.md` file exist? I can't find it, and the link 404s.